lib/utils.h

```diff
@@ -97,7 +98,7 @@ inline static bool isPrefixStringCharLiteral(const std::string &str, char q, con
 
 inline static bool isStringCharLiteral(const std::string &str, char q)
 {
-    static const std::vector<std::string> suffixes{"", "u8", "u", "U", "L"};
+    static const std::array<std::string, 5> suffixes{{"", "u8", "u", "U", "L"}};
```

Comment:
hmm.. shouldn't we be able to construct the array without extra {}?

    static const std::array<std::string, 5> suffixes{"", "u8", "u", "U", "L"};

Reply:
We are, but not in all cases. I removed the unnecessary extra ones.